src/client/components/Sort/Sort.component.js

```diff
@@ -2,65 +2,100 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './Sort.styles.css';
 
-export default function Sort({ products, setSortedProducts }) {
-  const [option, setOption] = React.useState();
-  function sortProducts(opt) {
-    if (opt === 'AlphabeticallyAZ') {
-      return products.sort((a, b) => a.name.localeCompare(b.name));
-    }
-    if (opt === 'AlphabeticallyZA') {
-      return products.sort((a, b) => b.name.localeCompare(a.name));
-    }
-    if (opt === 'PriceDescending') {
-      return products.sort((a, b) => a.price - b.price);
+const SORT_OPTION_MAPPING = {
```

Comment:
Out of scope for this PR:
Like we spoke, this component owns the format of the data AND the data itself.
If we wanted it to be more re-useable it would be better if it only owned the format of the data.
Imagine an interface like this:
```
<SortOrderSelector
  currentSortOrder={1}
  sortOptions={
    [
      { value: 0, field: 'price', direction: 'asc', label: 'Price (low to high)' }, 
      { value: 1, field: 'name', direction: 'asc', label: 'Name A-Z' }
    ]
  }
  onSortOrderChanged={(newSortOrder) => setSortOrder(newSortOrder)}
/>
```

In this way, just like the ProductsView, it does not care about which SortOrders are possible, it just shows a list of the possible options and lets the LandingPage know which is the next sort order to use when it changes. This way it can be used to sort anything.